ts-service/package.json

```diff
@@ -11,12 +11,14 @@
     "author": "simonpiscitelli@gmail.com",
     "private": true,
     "dependencies": {
-        "@yupana/core-lib-poc": "^0.1.0",
-        "typescript": "^3.3.3333",
-        "tslib": "^1.9.3"
+        "@yupana/class-generator": "^0.1.3",
+        "@yupana/framework-service-types": "file:../../yupana-service-lib",
```

Comment:
Should it be a separate package or included into framework-types ?

At the moment it exports only 3 types from https://github.com/YupanaInc/Yupana-Framework/blob/build/3.0.0/server/lib/services/server.js : 
```
export {Task};
export {Service};
export {ServicePackage};
```

Reply:
Maybe framework-types... ?

Reply:
i agree — the naming of the npm packages we're publishing aren't... "optimal"...

- the `@yupana/class-generator` does not generate any classes, it's the base data types (`YTClass`, `YTslot`, `YTarray`, etc.)
- there's `@yupana/framework-types` already, which are the interface definitions for the likes of `YSession`, `YQuery`, `YMongoDB`, `YRole` and such — core framework types?

but these `@yupana/framework-service-types` seem to be accurate — they are types that only matter to services, they cannot be used elsewhere...

i suggest dropping the "framework" from the package names — and...
- rename `@yupana/class-generator` to `@yupana/base-types`
- rename `@yupana/framework-types` to `@yupana/core-types`
- and rename this latest `@yupana/framework-service-types` to `@yupana/service-types`
 (not sure how this works from an npm standpoint, must it be a new package or does it support renaming?)

Reply:
Yeah, I like single-purpose packages like `@yupana/service-types` myself. They seem idiomatic to npm. I am just not quite sure about maintenance cost. It depends on core types, so all changes there must be propagated upstream. 
Including service types into the core will significantly simplify updates, and we gonna have a lot of them at first.

Reply:
Added to the core https://github.com/YupanaInc/yupana-core-lib/pull/3